package.json

```diff
@@ -38,6 +38,9 @@
     "lint:fix": "eslint . --fix",
     "mozilla-lint": "addons-linter dist/firefox",
     "watch": "cross-env METAMASK_ENV=test mocha --watch --require test/setup.js --reporter min --recursive \"test/unit/**/*.js\" \"ui/app/**/*.test.js\"",
+    "devtools:react": "react-devtools",
+    "devtools:redux": "remotedev --hostname=localhost --port=8000",
+    "devtools:ui": "concurrently -k -n react,redux npm:devtools:react devtools:redux",
```

Comment:
```suggestion
    "devtools:ui": "concurrently -k -n react,redux npm:devtools:react npm:devtools:redux",
```

Reply:
I'm belatedly realizing now that `concurrently` isn't even listed as a dependency yet - I suppose that's what motivated the other PR that adds it!

Reply:
Yeah I'll drop this script entry for now